src/services/itemMembership/index.ts

```diff
@@ -36,16 +38,14 @@ const plugin: FastifyPluginAsync = async (fastify) => {
         fastify.register(fastifyCors, fastify.corsPluginOptions);
       }
 
-      // if (WEBSOCKETS_PLUGIN) {
-      //   registerItemMembershipWsHooks(
-      //     websockets,
-      //     runner,
-      //     itemsDbService,
-      //     dbService,
-      //     taskManager,
-      //     db.pool,
-      //   );
-      // }
+      if (WEBSOCKETS_PLUGIN) {
+        registerItemMembershipWsHooks(
+          buildRepositories(),
```

Comment:
I guess it's fine to pass the repos for now. In the items, I've called `buildRepositories` in each function, but the idea is to replace the injection by letting each service resolve its own dependencies anyway.